src/alexandria3k/file_pubmed_cache.py

```diff
@@ -41,9 +42,11 @@ def read(self, path):
         if path == self.cached_path:
             return self.cached_data
 
-        # print(f"READ FILE {path}")
         with gzip.open(path, "rb") as uncompressed_file:
-            file_content = uncompressed_file.read()
+            try:
+                file_content = uncompressed_file.read()
+            except BadGzipFile:
+                file_content = ""

```

Comment:
Should you emit a warning or raise an exception here? Why the silent failure?

Reply:
A warn message is now displayed, the main reason for this try, except is that while downloading the Pubmed files from the FTP server some hidden files were also downloaded. These files start with `._` and other than that have the same filenames as the other files.

Reply:
Maybe then only process files that match a good file pattern.